scenarios/microsoft-azure-arcbox-v1/arcbox-overview.md

```diff
@@ -0,0 +1,58 @@
+# Jumpstart ArcBox
+
+## In This Guide
+
+Learn more about ArcBox options, features, and deployment methods.
```

Comment:
You no longer need this document. Arc box has moved to the /solutions folder.. this is not necessary.